private-templates-service/client/src/store/currentTemplate/reducers.ts

```diff
@@ -49,6 +54,7 @@ export function currentTemplateReducer(state = initialState, action: CurrentTemp
         templateJSON: action.templateJSON,
         templateName: action.templateName,
         sampleDataJSON: action.sampleDataJSON,
+        version: action.version,
```

Comment:
I don't believe `action.version` gets updated in the action `receiveNewTemplateUpdate`. Nevertheless, if a new template is being POSTed to the backend, we know version is going to be "1.0".

Reply:
Trueeeeee. Removed.